src/github/client/github-client-server.test.ts

```diff
@@ -79,7 +79,7 @@ describe("GitHub Client", () => {
 		const appTokenHolder = new AppTokenHolder((installationId: InstallationId) => {
 			switch (installationId.githubBaseUrl) {
 				case gheUrl:
-					return fs.readFileSync(envVars.PRIVATE_KEY_PATH, { encoding: "utf8" });
+					return Promise.resolve(fs.readFileSync(envVars.PRIVATE_KEY_PATH, { encoding: "utf8" }));
```

Comment:
Maybe don't need to write `Promise.resolve(`

Reply:
Signature of KeyLocator has been changed, That's why need to update this one.

Reply:
use async instead.  You shouldn't have to explicitly call `promise.resolve` or `promise.reject`